### PR TITLE
Capture more fixes on disconnect and status

### PR DESCRIPTION
**Pull request by sidpremkumar**
## Description of the change

This PR ensures that status and disconnect are working as intended. 

## Relevant release note information

Release Notes:

## Related JIRA tickets

Relates to JIRA: CWC-XXX

## Have you considered the security impacts?

Does this PR have any security impact?

- [ ] Yes
- [X] No

If yes, please explain:

### CODE REVIEW

**Review by EthanHeilman — APPROVED** [review]
These changes look good to merge. I looked up `killPortProcess` and if I read the code correctly it just kills any process which is using that port.  Kill other processes on a system that we didn't start is pretty aggressive. Is there a better way?

File: src/handlers/status/status.handler.ts [hunk 1]
```diff
@@ -49,7 +49,6 @@ async function webStatusHandler(
             await killPortProcess(webConfig['localPort']);
 
             configService.setWebConfig(webConfig);
-            await cleanExit(0, logger);
```
Comment (lgmugnier) [inline src/handlers/status/status.handler.ts:52]: why don't we need the cleanExit anymore?

Comment (sidpremkumar) [reply src/handlers/status/status.handler.ts:52]: incase the user wants to see more than 1 status: 
2022-01-11 at 5 01 PM

Comment (sidpremkumar) [reply src/handlers/status/status.handler.ts:52]: https://github.com/bastionzero/zli/pull/283/files#diff-10b5461068de4b9dc04c80862ff4b783437bf57e3009ff7b8741519f6856dfadL16-L24

**Comment by sidpremkumar** [comment]
> These changes look good to merge. I looked up `killPortProcess` and if I read the code correctly it just kills any process which is using that port. Kill other processes on a system that we didn't start is pretty aggressive. Is there a better way?

Yeah this is the implementation we've decided to go with, we need to eventually reevaluate if this is the best way to do things, but we save the port info, so we are only killing a process on a port that we know about. 

Tbh I do not like the way we manage daemons that are spawned and would like to re-design this part at a later date.

**Pull request merged**